### PR TITLE
Add Chinese translations

### PR DESCRIPTION
**Pull request by kinosang**
* [X] Add Simplified Chinese translations
* [x] Add Traditional Chinese translations

### CODE REVIEW

**Comment by kevinchalet** [comment]
Thanks a lot for your PR! 👏

**Comment by kevinchalet** [comment]
I see you've opted for `zh-CN` (simplified Chinese -> Mainland China) and `zh-TW` (traditional Chinese -> Taiwan).

Do you strongly feel we need to keep the region part or could we remove it and use `zh-Hans` (simplified Chinese) and `zh-Hant` (traditional Chinese) instead?

**Comment by kinosang** [comment]
@kevinchalet yep, I think `zh-Hans` and `zh-Hant` are better. But it seems Asp.Net Core itself uses `zh-CN` and `zh-TW` for l10n resources (or I got it wrong).

In short, I've rename the `xlf`s to use the neutral culture name.

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hans.xlf [hunk 1]
```diff
@@ -0,0 +1,602 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="zh-CN" original="../OpenIddictResources.resx">
```
Comment (kevinchalet) [inline src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hans.xlf]: Please replace `zh-CN` by `zh-Hans` to match the file name.

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hant.xlf [hunk 2]
```diff
@@ -0,0 +1,602 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="zh-TW" original="../OpenIddictResources.resx">
```
Comment (kevinchalet) [inline src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hant.xlf]: Same: replace `zh-TW` by `zh-Hant`.

**Review by kevinchalet — APPROVED** [review]

File: src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hans.xlf (hunk 1 above)
Comment (kinosang) [reply src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.zh-Hans.xlf]: uh, I missed it...

**Comment by kevinchalet** [comment]
Could you also please update `XlfLanguages` to ensure the `zh-Hans` and `zh-Hant` translations are correctly tracked?

https://github.com/openiddict/openiddict-core/blob/dev/Directory.Build.props#L12

**Comment by kinosang** [comment]
@kevinchalet all done :)

**Comment by kevinchalet** [comment]
Merged, thanks a lot! 👏

**Pull request merged**